### ISSUE TITLE
RTCIceCandidate: relatedAddress Examples section error

### ISSUE THREAD

**Issue by ciyoe** [mdn/content#33227]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/relatedAddress

### What information was incorrect, unhelpful, or incomplete?

```js
switch (candidate.type) {
  case "host":
    console.log(`Host candidate's IP address is ${candidate.ip}`);
    break;
  case "srflx":
    console.log(
      `Server reflexive candidate's base address is ${candidate.relatedAddress}; reachable at ${candidate.ip}`,
    );
    break;
  case "prflx":
    console.log(
      `Peer reflexive candidate's base address is ${candidate.relatedAddress}; reachable at ${candidate.ip}`,
    );
    break;
  case "relay":
    console.log(
      `Relay candidate's address assigned by the TURN server is ${candidate.relatedAddress}; reachable at ${candidate.ip}`,
    );
    break;
}
```

the candidate has no `ip` property but `address` in [WebRTC Candidate's Attributes](https://w3c.github.io/webrtc-pc/#attributes-4)

### What did you expect to see?

```js
switch (candidate.type) {
  case "host":
    console.log(`Host candidate's IP address is ${candidate.address}`);
    break;
  case "srflx":
    console.log(
      `Server reflexive candidate's base address is ${candidate.relatedAddress}; reachable at ${candidate.address}`,
    );
    break;
  case "prflx":
    console.log(
      `Peer reflexive candidate's base address is ${candidate.relatedAddress}; reachable at ${candidate.address}`,
    );
    break;
  case "relay":
    console.log(
      `Relay candidate's address assigned by the TURN server is ${candidate.relatedAddress}; reachable at ${candidate.address}`,
    );
    break;
}
```

### Do you have any supporting links, references, or citations?

 [WebRTC Candidate's Attributes](https://w3c.github.io/webrtc-pc/#attributes-4)

### Do you have anything more you want to share?

also in 
[relatedAddress](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/relatedAddress),
[relatedPort](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/relatedAddress)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/rtcicecandidate/relatedaddress`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/relatedAddress
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/rtcicecandidate/relatedaddress/index.md
* Last commit: https://github.com/mdn/content/commit/592f6ec42e54981b6573b58ec0343c9aa8cbbda8
* Document last modified: 2023-07-25T00:27:48.000Z

</details>

**Comment by hamishwillee** [comment]
Closing, as fixed by your PR https://github.com/mdn/content/pull/33229

FYI, if you create a PR to fix an issue, in the PR description you can add a line `Fixes #33229` and it will automatically close the issue when the PR is merged.

**Comment by ciyoe** [comment]
ths for your tips
